### ISSUE TITLE
After conversion, V2 version of page still showing on View tab

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#5254]
After converting a node using the conversion tool, v3 drafts and published pages currently showing v2 content when you view them. You can only see the changes when viewing content in Layout Builder.

This may be related to the tempstore or needing a cache clear.

# Proposed solution
Investigate why this is happening. Does it happen to all nodes or just some? Is it because some nodes have data in the tempstore already?

**Comment by briand44** [comment]
Grooming notes: Attempt to recreate the problem, verify if it happens always or certain instances, try clearing cache? is it a tempstore issue?

**Comment by bspeare** [comment]
Running p2lb on nodes for hawkeyemarchingband.uiowa.edu, the converted version is available under "Latest Version".  That page needs to be moved out of Draft to Published to view the layout builder page.   Is that the expected behavior?  I imagine we want to set the latest version to be published after the conversion?

**Comment by cory-skeers** [comment]
> Running p2lb on nodes for hawkeyemarchingband.uiowa.edu, the converted version is available under "Latest Version". That page needs to be moved out of Draft to Published to view the layout builder page. Is that the expected behavior? I imagine we want to set the latest version to be published after the conversion?

Ahhh...I think the idea was that if this is being run in a prod site, we'd create a new draft to be reviewed before replacing the existing page just in case, but if it leads to confusion or seemingly weird extra steps, maybe it would be better to set it as the new published version. There would still be the previous revision to roll back to, if needed.

**Comment by pyrello** [comment]
>> Running p2lb on nodes for hawkeyemarchingband.uiowa.edu, the converted version is available under "Latest Version". That page needs to be moved out of Draft to Published to view the layout builder page. Is that the expected behavior? I imagine we want to set the latest version to be published after the conversion?
>
> Ahhh...I think the idea was that if this is being run in a prod site, we'd create a new draft to be reviewed before replacing the existing page just in case, but if it leads to confusion or seemingly weird extra steps, maybe it would be better to set it as the new published version. There would still be the previous revision to roll back to, if needed.

We absolutely do not want these to be set to published upon completion of the action. That means that people will be publishing new versions of their pages without getting a chance to review them. I think that would be a dealbreaker for the brand site.

Here is an example of a page on sandbox where I have saved a new draft with changes to the layout: https://sandbox.prod.drupal.uiowa.edu/node/471/layout. The change I made is to remove a slide from the slider. When you visit the link, you can see that LB reflects these changes, while the published page does not: https://sandbox.prod.drupal.uiowa.edu/joes-example-page. That is the state we are trying to get to. There is some finishing step that is not happening and that is what this issue is meant to figure out.

**Comment by bspeare** [comment]
They way it is currently working seems correct.  If you convert a page, that v3 page is set as a draft and continues to display the v2 page until you publish the v3 page.  @pyrello what step are you looking for in between?

**Comment by bspeare** [comment]
I think this is working as expected and this issue can be closed.   @pyrello had a suggestion below that we might want to consider to clear up confusion between the paragraphs and layout builder editing options:

I think we will want to have some logic that checks to see if the conversion is run (that a value has been set on the conversion tracking field) and disable the paragraphs field on the edit page and include a note that the page either needs to be edited through layout builder or reverted.

**Comment by bspeare** [comment]
Looks like this still is an issue?  I synced this again and I'm seeing the problem now.

**Comment by pyrello** [comment]
> Looks like this still is an issue? I synced this again and I'm seeing the problem now.

Can you clarify what the problem actually is? As currently worded, the issue is a bit ambiguous. It was my best attempt at trying to explain it, but I did not do any actual investigation when I wrote it up.

**Comment by bspeare** [comment]
This issue I just encountered after converting a page:
- v2 version shows up as "Latest version" and publishing it keeps v2 version rendered
- v3 version is available under "Layout" but won't appear if you publish it. 

To get it to work, I had to navigate to `/admin/structure/types/manage/page/display` and check the "Allow each content item to have its layout customized" checkbox.   Once that was checked, the v3 version worked.
